### DISCUSSION TITLE
OSU! fails to play any audio

### DISCUSSION THREAD

**Discussion by walksanatora** [ppy/osu#22316] [Q&A] [answered]
### Type

Game behaviour

### Bug description

i launch OSU and there is no audio, i am running linux, Arch to be specific

### Screenshots or videos

![Screenshot_2023-01-20_22-30-14](https://user-images.githubusercontent.com/47339836/213846971-5cc12811-2ff7-4e4a-8797-861fafe09e6b.png)

### Version

2022.1228.0.r689.2075583313-1

### Logs

[database.log](https://github.com/ppy/osu/files/10471653/database.log)
[input.log](https://github.com/ppy/osu/files/10471654/input.log)
[network.log](https://github.com/ppy/osu/files/10471655/network.log)
[performance.log](https://github.com/ppy/osu/files/10471656/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10471657/runtime.log)

**Comment by cfgexe** [comment]
Something is hogging your audio output.

**Comment by walksanatora** [comment]
but here is the think i can launch any applicaation after osu and it works
unless PulseAudio is doing something funky i dont know
because the only active input is system sounds and that is muted
and none of the other application on my computer behave like this
and just checked, OSU is not even showing up as a audio source in pavucontrol

**Comment by DarioTD** [comment]
I reproduced the "code 46: Busy" issue by choosing "HDA Intel PCH" (which is being handled by pipewire in my case)
Choose "Pulseaudio Sound Server" in Settings -> Output Devices to solve it
In my case I choosed "Pipewire Sound Server" which solved it

**Comment by frenzibyte** [comment, marked as answer]
Please search for existing discussions, I'm sure one thread already has what you're looking for. Take a look at https://github.com/ppy/osu/discussions/21470 for example.

**Reply by walksanatora** [reply]
also the option to set audio output to `Pulseaudio Sound Server` fixed it
